llvm/lib/Target/RISCV/RISCV.td

```diff
@@ -170,6 +170,24 @@ def HasStdExtZvamo : Predicate<"Subtarget->hasStdExtZvamo()">,
                               AssemblerPredicate<(all_of FeatureExtZvamo),
                               "'Zvamo'(Vector AMO Operations)">;
 
+// CORE-V extensions
+// These extensions can either be enabled individually with their extension
+// name or all together with `+xcorev`
+def FeatureExtCoreV
+    : SubtargetFeature<"xcorev", "HasStdExtCoreV", "true",
```

Comment:
This shouldn't be called `HasStdExtCoreV`... `CoreV` is not a standard extension. I would follow a naming convention of `HasExtXCoreV`